graphql-client-codegen/build/generatedAntlr/com/martmists/multiplatform/graphql/codegen/GraphQLLexer.kt

```diff
@@ -0,0 +1,149 @@
+// Generated from C:/Users/micha/IdeaProjects/multiplatform-everything/graphql-client-codegen/antlr/GraphQL.g4 by ANTLR 4.13.1
```

Comment:
Ideally these would only specify `GraphQL.g4`

Reply:
I am not sure what I would do about this, since it would be overwritten everytime

Reply:
Is there no setting for this in the kotlin antlr fork?

Reply:
I wasn't able to find one